docs/userguides/development.md

```diff
@@ -0,0 +1,116 @@
+# Developing a Silverback Application
+
+Silverback applications are excellent for use cases that involve continuously monitoring and responding to on-chain events, such as newly confirmed blocks or contract event logs.
+
```

Comment:
Can this be added to the main README.md file?
I landed on the repo trying to figure out what was silverback for

Reply:
Done, thanks for the suggestion!